modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/openapi/contributor/ObjectEntryOpenAPIContributor.java

```diff
@@ -98,6 +98,14 @@ public void contribute(OpenAPI openAPI, OpenAPIContext openAPIContext)
 		Paths paths = openAPI.getPaths();
 
 		for (String key : new ArrayList<>(paths.keySet())) {
+			if (_objectDefinition.isRootDescendantNode() &&
```

Comment:
Currently, we are including [here](https://github.com/liferay/liferay-portal/blob/9eda8047441a650fab4b111de58595c260b8025d/modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java#L251) the operationIds of the operations that are being filtered when a request is performed. We would prefer you to include the operationId in the root descendant use case instead of modifying the OpenAPIContributor.

For example:

```
private void _excludeMethods(
		ObjectDefinition objectDefinition,
		ObjectScopeProvider objectScopeProvider) {

				[...]

				if ((!groupAware && hasScope) ||
					(groupAware && !hasScope &&
					 !value.startsWith("/{objectEntryId}")) ||
					(objectDefinition.isRootDescendant() &&
					 value.endsWith("/permissions"))) {

					excludedOperationIds.add(method.getName());
				}
			}
[...]
```

Reply:
Furthermore, we would be automatically filtering also the queries in GraphQL as the functionality is covered by the `excludedOperationIds` parameter

Reply:
@ccorreagg I updated the PR with the suggested changes. Thanks!